packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -328,19 +315,27 @@ class WebpackCLI {
             ) {
                 options.mode = process.env.NODE_ENV;
             }
-
-            if (args.name) {
-                options.name = args.name;
-            }
-
-            if (typeof args.stats !== 'undefined') {
-                options.stats = args.stats;
+            if (args.entry) {
```

Comment:
Why it here?

Reply:
some bugs in cli.processArguments, it's not able to handle multiple flags, will check later

Reply:
We need to fix it here